examples/with-typescript-graphql/pages/index.tsx

```diff
@@ -1,20 +1,25 @@
+import React from 'react'
 import Link from 'next/link'
 import { useViewerQuery, ViewerDocument } from '../lib/viewer.graphql'
 import { initializeApollo } from '../lib/apollo'
 
 const Index = () => {
   const { data } = useViewerQuery()
-  const { viewer } = data!
 
-  return (
-    <div>
-      You're signed in as {viewer.name} and you're {viewer.status} go to the{' '}
-      <Link href="/about">
-        <a>about</a>
-      </Link>{' '}
-      page.
-    </div>
-  )
+  if (data) {
+    const { viewer } = data!
+    return (
+      <div>
+        You're signed in as {viewer.name} and you're {viewer.status} go to the{' '}
+        <Link href="/about">
+          <a>about</a>
+        </Link>{' '}
+        page.
+      </div>
+    )
+  }
+
+  return <div>Loading...</div>
```

Comment:
Why did you add a loading here? in this page the data will always be defined, otherwise it's broken.

Reply:
True. I'll make the test work assuming the data always exists.